Common/DTO/RestRequests/CreateUserRequest.cs

```diff
@@ -9,5 +9,7 @@ public class CreateUserRequest
         public string FirstName { get; set; }
         public string LastName { get; set; }
         public DateTime Birthday { get; set; }
+        public byte[] Avatar { get; set; }
+        public string AvatarType { get; set; }

```

Comment:
Well, I didn't understand that it's a file type without seeng further usage, mb we can rename this property?

Reply:
Is ImageType ok?